src/Compilers/Server/VBCSCompiler/CompilerRequestHandler.cs

```diff
@@ -62,6 +62,11 @@ internal sealed class CompilerServerHost : ICompilerServerHost
 
         public ICompilerServerLogger Logger { get; }
 
+        /// <summary>
+        /// A persistent cache that can store generator drivers in order to enable incrementalism across builds.
```

Comment:
>persistent [](http://example.com/codeflow?start=14&length=10)

Is the cache lifetime the same as the `CompilerServerHost`? If so, consider removing "persistent".